openpype/hosts/tvpaint/plugins/load/load_workfile.py

```diff
@@ -0,0 +1,89 @@
+import getpass
+import os
+
+from avalon.tvpaint import lib, pipeline, get_current_workfile_context
+from avalon import api, io
+import openpype
+
+
+class LoadWorkfile(pipeline.Loader):
+    """Load workfile."""
+
+    families = ["workfile"]
+    representations = ["tvpp"]
+
+    label = "Load Workfile"
+
+    def load(self, context, name, namespace, options):
+        filepath = self.fname.replace("\\", "/")
+
+        if not os.path.exists(filepath):
+            raise FileExistsError(
+                "The loaded file does not exist. Try downloading it first."
+            )
+
+        george_script = "tv_LoadProject '\"'\"{}\"'\"'".format(
+            filepath
+        )
+        lib.execute_george_through_file(george_script)
+
+        # Save workfile.
+        host = api.registered_host()
+
+        project = io.find_one({
+            "type": "project"
+        })
+        context = get_current_workfile_context()
+        template_key = openpype.lib.get_workfile_template_key_from_context(
+            context["asset"],
+            context["task"],
+            host,
+            project_name=project["name"]
+        )
+        anatomy = openpype.Anatomy(project["name"])
+        data = {
+            "project": {
+                "name": project["name"],
+                "code": project["data"].get("code")
+            },
+            "asset": context["asset"],
+            "task": context["task"],
+            "version": 1,
+            "user": getpass.getuser(),
+            "root": {
+                template_key: anatomy.roots[template_key]
+            },
```

Comment:
Just pass `anatomy.roots`.
```suggestion
            "root": anatomy.roots,
```